net-misc/hylafaxplus/hylafaxplus-7.0.2.ebuild

```diff
@@ -0,0 +1,160 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit pam toolchain-funcs
+
+MY_PN="${PN/plus/}"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Enterprise client-server fax package for class 1 and 2 fax modems"
+HOMEPAGE="https://hylafax.sourceforge.net"
+SRC_URI="mirror://sourceforge/hylafax/${MY_P}.tar.gz"
+
+SLOT="0"
+LICENSE="hylafaxplus"
+KEYWORDS="~amd64 ~x86"
+IUSE="html jbig lcms ldap mgetty pam"
+
+DEPEND="
+	app-text/ghostscript-gpl
+	media-libs/tiff:0[jbig?]
+	!net-dialup/mgetty[fax]
+	>=sys-libs/zlib-1.1.4
+	virtual/awk
+	virtual/jpeg:0
+	virtual/mta
+	jbig? ( media-libs/jbigkit )
+	lcms? ( media-libs/lcms )
+	ldap? (  net-nds/openldap )
+	mgetty? ( net-dialup/mgetty[-fax] )
+	pam? ( sys-libs/pam )
+"
+RDEPEND="${DEPEND}
+	!net-dialup/sendpage
+	net-mail/metamail
+"
+
+S="${WORKDIR}/${MY_P}"
+
+CONFIG_PROTECT="${CONFIG_PROTECT} /var/spool/fax/etc /usr/lib/fax"
+CONFIG_PROTECT_MASK="${CONFIG_PROTECT_MASK} /var/spool/fax/etc/xferfaxlog"
+
+PATCHES=(
+	"${FILESDIR}/ldconfig-patch"
+	"${FILESDIR}/${PN}-CVE-2020-1539x.patch"
```

Comment:
If we can fetch it once, it'll be mirrored. But `CVE-2020-1539x.patch` doesn't look like it's going to be updated ever as-is.

So yeah please try to find a downloadable link for it.
https://devmanual.gentoo.org/ebuild-maintenance/new-ebuild/index.html

> The Gentoo repository should only be used for storing .ebuild files, as well as any small companion files, such as patches or sample configuration files. These files should have a size of at most 20 KiB and must be placed in the mycat/mypkg/files directory, which can be referenced as ${FILESDIR} from within ebuilds. Larger patch files should be placed in your developer space on dev.gentoo.org instead, in order to minimize the repository size.

Reply:
This force-push uses the patch I'm hosting from my Github pages, I'm already doing something similar with `app-text/djvu`.